apps/ui/src/helpers/intercom.ts

```diff
@@ -0,0 +1,47 @@
+const APP_ID = import.meta.env.VITE_INTERCOM_APP_ID;
+
+export function startIntercom() {
+  if (!APP_ID || document.body.clientWidth < 544) return;
+
+  // @ts-ignore
+  window.intercomSettings = {
+    app_id: APP_ID
+  };
+  const w = window;
```

Comment:
Can't you remove all the @ts-ignore if you do this?
```suggestion
  const w: any = window;
```